mission/TAC_tasks/scripts/pipeline_following/README.md

```diff
@@ -0,0 +1,8 @@
+## Futher improvements
+- Currently the the z value of the goal position is set to the same as the position of the drone. This should be improved. Maybe it shouldnt be updated at each iteration?
```

Comment:
The z value should be held at the initialization value, as updating the goal with the current z value may lead to undefined behaviour.

Reply:
initialized the z-value before the loop